README.md

```diff
@@ -119,6 +119,39 @@ Once in the web app:
 * Explore citations and sources
 * Click on "settings" to try different options, tweak prompts, etc.
 
+## Automate deployment
+
+You can use GitHub actions or Azure DevOps to deploy this demo. For example, after pushing changes to the repo.
+
+There is a `.github` and `.azdo` folder which provides a pipeline (or workflow) definition (`azure-dev.yml`). These pipelines require some configuration for the provider (GitHub or Azure DevOps), for example, setting secrets or variables. You can use `azd` to create the configuration for the provider:
+
+For `GitHub`, run:
+> azd pipeline config
```

Comment:
I think using three backticks is nicer than > which I believe is for blockquoting.